modules/agar/src/main/ts/ephox/agar/api/Main.ts

```diff
@@ -58,7 +57,6 @@ export {
   Monitor,
   Mouse,
   NamedChain,
-  PhantomSkipper,
```

Comment:
Agar changelog entry for removing this? Also we should add a major version bump for agar to `versions.txt`.

Reply:
Oh right, I was only thinking of the TinyMCE changelog.

Reply:
The text file already had it, for the record, due to the katamari upgrade.

Reply:
Ah right I should have looked there first sorry :disappointed: